src/fontra/views/editor/panel-reference-font.js

```diff
@@ -320,6 +360,24 @@ export default class ReferenceFontPanel extends Panel {
       await fontItem.fontFace.load();
     }
     this.model.referenceFontName = fontItem.fontIdentifier;
+
+    if (Object.hasOwn(this.supportedLanguagesMemoized, fontItem.fontIdentifier)) {
```

Comment:
I think this is good enough in this case, and reads better (to a Python reader at least):
```suggestion
    if (fontItem.fontIdentifier in this.supportedLanguagesMemoized) {
```

Reply:
The font identifier contains a random UUID, so this is a moot point. Please use "in" like I suggested.

Reply:
Ok, I thought it is the font name.